source/base/quadrature_lib.cc

```diff
@@ -144,6 +144,113 @@ namespace internal
 } // namespace internal
 
 
+namespace internal
+{
+  namespace QGaussRadau
+  {
+    // Implementes lookup table after affine transformation
+    // original for [-1,1] listed on
+    // https://mathworld.wolfram.com/RadauQuadrature.html
+    std::vector<double>
+    get_left_quadrature_points(const unsigned int n)
+    {
+      std::vector<double> q_points(n);
+      switch (n)
+        {
+          case 1:
+            q_points[0] = 0.;
+            break;
+          case 2:
+            q_points[0] = 0.;
+            q_points[1] = 2. / 3.;
+            break;
+          case 3:
+            q_points[0] = 0.;
+            q_points[1] = (6. - std::sqrt(6)) * 0.1;
+            q_points[2] = (6. + std::sqrt(6)) * 0.1;
+            break;
+          default:
+            Assert(false, dealii::StandardExceptions::ExcNotImplemented());
+            break;
+        }
+      return q_points;
+    }
+
+    std::vector<double>
+    get_quadrature_points(const unsigned int                 n,
+                          ::dealii::QGaussRadau<1>::EndPoint ep)
+    {
+      std::vector<double> points(n);
+      std::vector<double> left_points = get_left_quadrature_points(n);
+      switch (ep)
+        {
+          case ::dealii::QGaussRadau<1>::left:
+            return left_points;
+          case ::dealii::QGaussRadau<1>::right:
+            for (unsigned int i = 0; i < n; ++i)
+              {
+                points[i] = 1. - left_points[i];
```

Comment:
Personally, I would prefer if the points were sorted. Could you flip the direction of the for loop?

Reply:
Done